build/build.xml

```diff
@@ -55,7 +81,7 @@
     <target name="buildSalesforceHybridSDK">
         <antcall target="buildLib">
             <param name="projectName" value="SalesforceHybridSDK" />
-            <param name="libName" value="SalesforceHybridSDK.framework" />
+            <param name="libName" value="libSalesforceHybridSDK.a" />
             <param name="headerDir" value="Headers/SalesforceHybridSDK" />
             <param name="schemeName" value="SalesforceHybridSDK" />
```

Comment:
The scheme name is probably going to be SalesforceHybridSDK-Static (or whatever we decide to call it)